test/org/sagebionetworks/bridge/services/AuthenticationServiceMockTest.java

```diff
@@ -0,0 +1,284 @@
+import org.sagebionetworks.bridge.models.accounts.UserSession;
+import org.sagebionetworks.bridge.models.studies.EmailTemplate;
+import org.sagebionetworks.bridge.models.studies.MimeType;
+import org.sagebionetworks.bridge.models.studies.Study;
+import org.sagebionetworks.bridge.models.studies.StudyIdentifierImpl;
+import org.sagebionetworks.bridge.models.subpopulations.SubpopulationGuid;
+import org.sagebionetworks.bridge.services.email.EmailSignInEmailProvider;
+import org.sagebionetworks.bridge.services.email.MimeTypeEmail;
+import org.sagebionetworks.bridge.validators.PasswordResetValidator;
+
+import com.google.common.collect.ImmutableMap;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AuthenticationServiceMockTest {
+    
+    private static final String STUDY_ID = "test-study";
+    private static final String CACHE_KEY = "email@email.com:test-study:signInRequest";
+    private static final String RECIPIENT_EMAIL = "email@email.com";
+    private static final String TOKEN = "ABC-DEF";
+    private static final SignIn SIGN_IN_REQUEST = new SignIn(STUDY_ID, RECIPIENT_EMAIL, null, null);
+    private static final SignIn SIGN_IN = new SignIn(STUDY_ID, RECIPIENT_EMAIL, null, TOKEN);
+    private static final SubpopulationGuid SUBPOP_GUID = SubpopulationGuid.create("ABC");
+    private static final ConsentStatus CONSENTED_STATUS = new ConsentStatus.Builder().withName("Name")
+            .withGuid(SUBPOP_GUID).withRequired(true).withConsented(true).build();
+    private static final ConsentStatus UNCONSENTED_STATUS = new ConsentStatus.Builder().withName("Name")
+            .withGuid(SUBPOP_GUID).withRequired(true).withConsented(false).build();
+    private static final Map<SubpopulationGuid, ConsentStatus> CONSENTED_STATUS_MAP = new ImmutableMap.Builder<SubpopulationGuid, ConsentStatus>()
+            .put(SUBPOP_GUID, CONSENTED_STATUS).build();
+    private static final Map<SubpopulationGuid, ConsentStatus> UNCONSENTED_STATUS_MAP = new ImmutableMap.Builder<SubpopulationGuid, ConsentStatus>()
+            .put(SUBPOP_GUID, UNCONSENTED_STATUS).build();
+    private static final CriteriaContext CONTEXT = new CriteriaContext.Builder()
+            .withStudyIdentifier(TestConstants.TEST_STUDY).build();  
+    private static final StudyParticipant PARTICIPANT = new StudyParticipant.Builder().build();
+
+    @Mock
+    private CacheProvider cacheProvider;
+    @Mock
+    private BridgeConfig config;
+    @Mock
+    private ConsentService consentService;
+    @Mock
+    private ParticipantOptionsService optionsService;
+    @Mock
+    private AccountDao accountDao;
+    @Mock
+    private ParticipantService participantService;
+    @Mock
+    private SendMailService sendMailService;
+    @Mock
+    private StudyService studyService;
+    @Mock
+    private PasswordResetValidator passwordResetValidator;
+    @Mock
+    private Study study;
+    @Mock
+    private Account account;
+    @Mock
+    private UserSession session;
```

Comment:
Similarly, study and session don't need to be mocked, and account can just use SimpleAccount.